heroku/cli
# refactor(redis-v5): Move 'redis:upgrade' to CLI

### Description
https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001axUyLYAU/view

This converts `redis:upgrade` to oclif and updates its associated tests.

### Testing

So this is a little tough because to test the successful case, you need an older version of Redis, which you can't have on a newly created addon install. The best I could do is just try to upgrade with a redis that is already at the newest version and compare with the production equivalent.

1) Run `./bin/run redis:upgrade -a APP` with this branch pulled down with an app that has redis on it to update its version
2) Verify the command updates the policy by running `heroku redis:info -a APP`
3) Compare the results with prod cli command and make sure things look 👍
